user_manager.py

```
@@ -0,0 +1,114 @@
+                id INTEGER PRIMARY KEY,
+                username TEXT,
+                password TEXT,
+                email TEXT,
+                created_at TEXT
+            )
+        ''')
+        self.connection.commit()
+
+    def hash_password(self, password):
+        # Simple MD5 hash for password
+        return hashlib.md5(password.encode()).hexdigest()
```

Ruff (0.14.10)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)